tools/calendar_generation.py

```diff
@@ -0,0 +1,61 @@
+import calendar
+import datetime
+
+
+class CalendarTime():
+
+    def __init__(self, monthName, currentYear, currentMonth):
+        self.monthname = monthName
+        self.year = currentYear
+        self.month = currentMonth
+
```

Comment:
I think that maybe adding calendar function into class would make it more consistent.

This class could be organised in that way:
__init__ -> initialize monthname, year and month with current time values
set_next_month -> Set class month to next moht
set_previous_month -> Set class month to previous month
etc.